### DISCUSSION TITLE
Error installing pwntools: 404 not found

### DISCUSSION THREAD

**Discussion by PascalHonegger** [Homebrew/discussions#2497] [Getting started]
I'm trying to install pwntools using brew but I'm getting the following error output:

```sh
user:~$ brew install pwntools
==> Downloading https://ghcr.io/v2/homebrew/core/pwntools/manifests/4.7.0
curl: (22) The requested URL returned error: 404                              

Error: pwntools: Failed to download resource "pwntools_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/pwntools/manifests/4.7.0
```

I believe this to be an issue with homebrew, but I don't understand the internals enough. Version 4.7.0 does exist as a release of PwnTools on GitHub, I don't know how to furhter address this issue.

**Comment by cho-m** [comment]
I think `pwntools` was accidentally updated to 4.7.0 without bottles being built in a PR in https://github.com/Homebrew/homebrew-core/commit/be475e0c30d8ef36f8b946c942b5976f1d068f21

The current bottle on GitHub is still 4.6.0 (rev 1): https://github.com/Homebrew/homebrew-core/pkgs/container/core%2Fpwntools while the formula shows 4.7.0, which is probably why it is returning a HTTP 404 Not Found error.

Ping @chenrui333 for any clarification on commit.

This may need a new PR to build bottles and review.

**Reply by WaasilaAsif** [reply]
Helloo
I think it is a simple onelne command issue.
I am testting it out on my machine and would like to fix it
